packages/edit-site/src/components/template-actions/rename-menu-item.js

```diff
@@ -67,7 +69,7 @@ export default function RenameMenuItem( { template, onClose } ) {
 			<MenuItem
 				onClick={ () => {
 					setIsModalOpen( true );
-					setTitle( template.title.rendered );
+					setTitle( template.title?.rendered || template.title );
```

Comment:
These changes are weird, I guess the consequence of using it with both `getEditedEntityRecord` and `getEntityRecord`?  or is it something else?

Reply:
Probably that(at least my first thoughts too). I'll check it now better.